client/src/site-search/index.scss

```
@@ -6,4 +6,21 @@
 
 .site-search {
   display: block;
+
+  article {
+    @media screen and (max-width: $screen-md) {
+      display: flex;
+      flex-direction: column;
+    }
+
+    .place {
```

Are there multiple `.place` elements?

```suggestion
    section.place {
```